### DISCUSSION TITLE
execute boot script

### DISCUSSION THREAD

**Discussion by AndreoBotelho** [micropython/micropython#16240] [Zephyr] [answered]
it is possible to have a boot script executing at boot time? I'm using  a external storage as /flash but it needs to be mounted with a script at boot.

**Comment by bidrohini** [comment, marked as answer]
It is possible to execute a script at boot time to mount your external storage as /flash.

**Reply by AndreoBotelho** [reply]
Thanks, I was having a lot of trouble to init my QSPI flash as storage but as long as I've done the drive /flash appeared in thonny, unfortunately thonny is not working fine with this build, the repl is crashing a lot and cant read files from storage, but now I can send files an test some scripts
